etc/config/c++.amazon.properties

```diff
@@ -2166,6 +2166,13 @@ libs.magic_enum.versions.trunk.path=/opt/compiler-explorer/libs/magic_enum/trunk
 libs.magic_enum.versions.073.version=0.7.3
 libs.magic_enum.versions.073.path=/opt/compiler-explorer/libs/magic_enum/v0.7.3/include
 
+libs.mfem.name=mfem
+libs.mfem.versions=master
```

Comment:
Can you change this version identifier to trunk? Just so it follows the rest of the libraries